src/core/BaseNode.js

```diff
@@ -245,5 +178,72 @@ export default Class(null, {
    * @method exit
    * @param {Tick} tick A tick instance.
    **/
-  exit: function(tick) {},
-});
+  exit(tick) {}
+};
+
+/**
+ * Node ID.
+ * @property {string} id
+ * @readonly
+ **/
+BaseNode.prototype.id = null;
```

Comment:
Good point. In that case I think we can drop it.

We have the same properties declared twice: https://github.com/Zielak/behavior3js/blob/35ed667ab0ea41c112d00e464aa783005f35b672/src/core/BaseNode.js#L29-L38 and https://github.com/Zielak/behavior3js/blob/35ed667ab0ea41c112d00e464aa783005f35b672/src/core/BaseNode.js#L184-L249

What do you think?